packages/aws-cdk-lib/custom-resources/README.md

```diff
@@ -824,3 +824,88 @@ new cr.AwsCustomResource(this, 'CrossAccount', {
   })]),
 });
 ```
+
+#### Custom Resource Config
+
+You can configure every CDK-vended custom resource in a given scope with `CustomResourceConfig`. 
+
+*Please understand that there is no guarantee the order in which Custom Resource Config is run. More than one Custom Resource Config of the same scope might result in conflict.
+
+The following example configures every custom resource in this CDK app to retain its logs for ten years:
+```ts
+    import { CustomResourceConfig } from 'aws-cdk-lib/custom-resources';
+
+    CustomResourceConfig.of(app).addLogRetentionLifetime(logs.RetentionDays.TEN_YEARS);
+
+    const stack = new cdk.Stack(app);
+
+    let websiteBucket = new s3.Bucket(stack, 'WebsiteBucket', {});
+    new s3deploy.BucketDeployment(stack, 's3deployNone', {
+      sources: [s3deploy.Source.jsonData('file.json', { a: 'b' })],
+      destinationBucket: websiteBucket,
+    });
+```
+The following example configures every custom resource in a nested stack to retain its log for ten years:
```

Comment:
```suggestion
`CustomResourceConfig` can modify resources across any scope. This example modifies every custom resource in two top-level stacks:
```

(move the top-level stacks example to be first, before the nested stack example) 

Can you also add unit tests for both of these cases?